### ISSUE TITLE
ISO9660 CD images support

### ISSUE THREAD

**Issue by XProger** [XProger/OpenLara#140]

**Comment by giuliozelante** [comment]
Actually, if you have the steam version you can just mount the file GAME.GOG as ISO in any mounting image program like Daemon Tools for Tomb Raider 1 it's **located in steamapps\common\Tomb Raider (I)**. When you try to select the image file just put the asterisk symbol (*****) in the filename search input box

**Comment by XProger** [comment]
yep, but using third party tools is not a user friendly approach )

**Comment by XProger** [comment]
@giuliozelante yep, but using third party tools is not a user friendly approach )
